### PR TITLE
asl 4574 named person training record

### PR DESCRIPTION
**Pull request by parvinba**

### CODE REVIEW

File: pages/profile/read/views/index.jsx [hunk 1]
```diff
@@ -78,9 +79,11 @@ const Index = ({
         allowedActions.includes('training.read') && (
           <section className="profile-section">
             <h3><Snippet>training</Snippet></h3>
-            <TrainingSummary certificates={certificates} />
+            <p>{instruction[0]}</p>
+            <p>{instruction[1]}</p>
```
Comment (jeff-horton-ho-sas) [inline pages/profile/read/views/index.jsx:83]:
```suggestion
            <Snippet>instruction</Snippet>
```

File: pages/training/dashboard/views/index.jsx [hunk 2]
```diff
@@ -12,11 +13,13 @@ export default function Training() {
         title={<Snippet>title</Snippet>}
         subtitle={`${profile.firstName} ${profile.lastName}`}
       />
+      <p>{instruction[0]}</p>
+      <p>{instruction[1]}</p>
```
Comment (jeff-horton-ho-sas) [inline pages/training/dashboard/views/index.jsx:17]:
```suggestion
      <Snippet>instruction</Snippet>
```

**Review by jeff-horton-ho-sas — CHANGES_REQUESTED** [review]
Looks good - some changes to fit in with other parts of the code base.

Let me know if you want a quick chat about how content objects/`<Snippet>` works?

File: pages/training/dashboard/content/index.js [hunk 3]
```diff
@@ -1,5 +1,7 @@
 module.exports = {
   title: 'Training record',
+  instruction: ['Update your record with the mandatory training modules required to support an establishment role application. Or request an exemption if you have equivalent training or professional experience which makes the training unnecessary.',
+    'You should not request an exemption for incomplete training as you still need to complete it.'],
```
[the remark this answers is not in the document]
Comment (parvinba) [reply pages/training/dashboard/content/index.js]: done

File: pages/profile/read/views/index.jsx (hunk 1 above)
Comment (parvinba) [reply pages/profile/read/views/index.jsx]: done

**Comment by parvinba** [comment]
@jeff-horton-ho-sas  - All suggestions were implemented, please rereview

**Pull request merged**